lib/resources/dstu2/allergy_intolerance.yaml

```diff
@@ -0,0 +1,219 @@
+---
+field_name_base_url: http://hl7.org/fhir/DSTU2/allergyintolerance-definitions.html#AllergyIntolerance
+fields:
+- name: resourceType
+  required: 'Yes'
+  cardinality: 1..1
+  type: string
+  description: The type of the FHIR resource.
+  example: AllergyIntolerance
+  note: resourceType must be AllergyIntolerance
+  url: http://hl7.org/fhir/DSTU2/json.html#resources
+
+- name: id
+  required: 'Yes'
+  cardinality: 0..1
+  type: id
+  description: The logical id of the resource to update.
+  example: '{"id": "123412"}'
+  note: The id value must match the AllergyIntolerance/&lt;id> value.
+  action: update
+
+- name: contained
+  required: 'No'
+  cardinality: 0..*
+  type: Resource
+  description: Contained, inline Resources.
+  example: |
+             Contained RelatedPerson
+               {"contained": [
+                  {"resourceType": "RelatedPerson",
+                   "id": "5366328",
+                   "relationship": {
+                      "coding": [
+                        {"system": "http://hl7.org/fhir/v3/RoleCode",
+                         "code": "SIGOTHR"}]}}]}
+
+             Contained Practitioner
+               {"contained": [
+                  {"resourceType": "Practitioner",
+                   "id": "123",
+                   "practitionerRole": [
+                     {"role": {
+                        "coding": [
+                          {"system": "http://hl7.org/fhir/v2/0286",
+                            "code": "RP"}]}}]}]
+  note: >
+          These should be either `Practitioner` or `RelatedPerson` resources that are referenced with the
```

Comment:
Cleaned up the type links and fixed RelatedPerson specifically:
https://github.com/cerner/fhir.cerner.com/commit/733a52e6680d4d66f1a62168e15df4eb4f48c112